packages/pwa/node.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
-module.exports = function installServiceWorke() {
+export default function installServiceWorke() {
```

Comment:
```suggestion
export default function installServiceWorker() {
```

We should fix this, while we're at it. :D